diff -Naurp gnupg-1.4.9/mpi/config.links gnupg-1.4.9.oden/mpi/config.links
--- gnupg-1.4.9/mpi/config.links	2008-03-25 11:15:47.000000000 +0100
+++ gnupg-1.4.9.oden/mpi/config.links	2008-12-30 19:44:21.000000000 +0100
@@ -220,6 +220,11 @@ case "${host}" in
 	path="powerpc64"
 	;;
 
+    powerpc64*-*-linux*)
+	echo '/* No assembler modules configured */' >>./mpi/asm-syntax.h
+	path=""
+	;;
+
     powerpc*-*-linux*)
 	echo '/* configured for powerpc/ELF */' >>./mpi/asm-syntax.h
 	echo '#define ELF_SYNTAX'               >>./mpi/asm-syntax.h
diff -Naurp gnupg-1.4.9/mpi/longlong.h gnupg-1.4.9.oden/mpi/longlong.h
--- gnupg-1.4.9/mpi/longlong.h	2007-10-23 11:15:14.000000000 +0200
+++ gnupg-1.4.9.oden/mpi/longlong.h	2008-12-30 19:44:21.000000000 +0100
@@ -923,7 +923,6 @@ extern USItype __udiv_qrnnd ();
 /* Powerpc 64 bit support taken from gmp-4.1.2. */
 /* We should test _IBMR2 here when we add assembly support for the system
    vendor compilers.  */
-#if 0 /* Not yet enabled because we don't have hardware for a test. */
 #if (defined (_ARCH_PPC) || defined (__powerpc__)) && W_TYPE_SIZE == 64
 #define add_ssaaaa(sh, sl, ah, al, bh, bl) \
   do {									\
@@ -936,7 +935,7 @@ extern USItype __udiv_qrnnd ();
     else								\
       __asm__ ("{a%I5|add%I5c} %1,%4,%5\n\t{ae|adde} %0,%2,%3"		\
 	     : "=r" (sh), "=&r" (sl)					\
-	     : "%r" (ah), "r" (bh), "%r" (al), "rI" (bl));		\
+	     : "r" (ah), "r" (bh), "%r" (al), "rI" (bl));		\
   } while (0)
 #define sub_ddmmss(sh, sl, ah, al, bh, bl) \
   do {									\
@@ -976,7 +975,6 @@ extern USItype __udiv_qrnnd ();
 #define SMUL_TIME 14  /* ??? */
 #define UDIV_TIME 120 /* ??? */
 #endif /* 64-bit PowerPC.  */
-#endif /* if 0 */
 
 /***************************************
  **************  PYR  ******************
