diff -Naur gnupg-2.4.0/doc/examples/systemd-user/gpg-agent.service gnupg-2.4.0.tpg/doc/examples/systemd-user/gpg-agent.service
--- gnupg-2.4.0/doc/examples/systemd-user/gpg-agent.service	2023-02-25 16:18:59.524455099 +0000
+++ gnupg-2.4.0.tpg/doc/examples/systemd-user/gpg-agent.service	2023-02-25 16:21:43.217202436 +0000
@@ -4,5 +4,6 @@
 Requires=gpg-agent.socket
 
 [Service]
-ExecStart=/usr/bin/gpg-agent --supervised
+EnvironmentFile=-/etc/sysconfig/gnupg
+ExecStart=/usr/bin/gpg-agent --supervised --no-allow-external-cache $GPGAGENT_PARAMS
 ExecReload=/usr/bin/gpgconf --reload gpg-agent
